integration-test/tests/integration_test.rs

```diff
@@ -19,16 +19,9 @@
 
 extern crate foundry_integration_test as test_common;
```

Comment:
"Move get_latest_block() inside the crate" commit does not move get_latest_block.  it removed it.

Reply:
Oh. My mistake. I'll fix.